app/assets/stylesheets/_buttons.scss

```diff
@@ -1,265 +1,159 @@
-    background-color: transparent;
-    color: $white;
-    &:hover {
-        color: $white;
-        background-color: rgba(0, 0, 0, 0.4);
-    }
+  @extend .button, .large;
+  @include transition-all(100ms);
+  border-radius: $global-radius;
+  border: 2px solid $white;
+  font-weight: 500;
+  background-color: transparent;
+  color: $white;
+
+  &:hover {
+    background-color: $white;
+    color: $charcoal;
+  }
 }
 
 a.transparent-success-button {
-    @extend .button;
-    border-radius: $global-radius;
-    border: 1px solid $charcoal;
-    background-color: transparent;
-    color: $charcoal;
-    &:hover {
-        color: $medium-green;
-        border: 1px solid $medium-green;
-    }
+  @extend .button;
+  border-radius: $global-radius;
+  border: 1px solid $charcoal;
+  background-color: transparent;
+  color: $charcoal;
+
+  &:hover {
+    color: $medium-green;
+    border: 1px solid $medium-green;
+  }
 }
 
 a.success-button, input.success-button {
-    @extend .button;
-    border-radius: $global-radius;
-    border: 1px solid $white;
-    color: $white;
-    background-color: $medium-green;
-    &:hover {
-        color: $white;
-        background-color: scale-color($medium-green, $lightness: 10%);
-    }
-
-}
+  @extend .button;
+  border-radius: $global-radius;
+  border: 1px solid $white;
+  color: $white;
+  background-color: $medium-green;
 
-.tabs {
-    position: relative;
-    overflow: hidden;
-    margin: 0 auto;
-    width: 100%;
-    font-weight: 300;
-    font-size: 1.25em;
-        ul {
-            text-align: center;
-            @include display-flex();
-            @include flex-flow();
-            position: relative;
-            margin: 0 auto;
-            padding: 0;
-            max-width: 1200px;
-            list-style: none;
-            -ms-box-orient: horizontal;
-            -ms-box-pack: center;
-            li {
-                @include flex();
-                position: relative;
-                z-index: 1;
-                display: block;
-                margin: 0;
-                text-align: center;
-                a {
-                    position: relative;
-                    display: block;
-                    overflow: hidden;
-                    text-overflow: ellipsis;
-                    white-space: nowrap;
-                    line-height: 2.5;
-                }
-            }
-        }
+  &:hover {
+    color: $white;
+    background-color: scale-color($medium-green, $lightness: 10%);
+  }
 }
 
 @mixin cubic-bezier-color() {
-	-webkit-transition: color 0.3s cubic-bezier(0.7,0,0.3,1);
-	transition: color 0.3s cubic-bezier(0.7,0,0.3,1);
+  -webkit-transition: color 0.3s cubic-bezier(0.7,0,0.3,1);
+  transition: color 0.3s cubic-bezier(0.7,0,0.3,1);
 }
 
 @mixin cubic-bezier-transform() {
-	-webkit-transition: transform 0.3s cubic-bezier(0.7,0,0.3,1);
-	transition: transform 0.3s cubic-bezier(0.7,0,0.3,1);
+  -webkit-transition: transform 0.3s cubic-bezier(0.7,0,0.3,1);
```

Comment:
Commas in function arguments should be followed by a single space<br>Avoid vendor prefixes.